src/App.tsx

```diff
@@ -1,17 +1,131 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
+import { Todo } from './types/Todo';
+
+const preparedTodos: Todo[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [isTitle, setIsTitle] = useState(true);
+  const [userName, setUserName] = useState('');
+  const [isUser, setIsUser] = useState(true);
+  const [copyTodos, setTodos] = useState(preparedTodos);
+
+  const handleTitle = (value: string) => {
+    const newTitle = value.replace(/[^\s\dA-ZА-Я.]/gi, '');
```

Comment:
it is not a good practice to change title without user's approve. Let's show message of invalid data instead)